src/elexclarity/client.py

```diff
@@ -92,29 +88,9 @@ def get_settings(self, electionid, statepostal, **kwargs):
         resp = self.make_request(f"{statepostal}/{electionid}/{current_ver}/json/en/electionsettings.json", **kwargs)
         return resp.json()
 
-    def get_counties(self, electionid, statepostal, **kwargs):
-        """
-        :param electionid: the election ID
-        :type electionid: str
-        :keyword statepostal: the state(s) to query
-        :type statepostal: str or tuple(str)
-
-        :param kwargs: other keyword args passed to underlying requests call
-        :return: the response json
-        :rtype: str
-
-        """
-        summary = self.get_settings(electionid, statepostal)
-        raw_counties = summary.get("settings", {}).get("electiondetails", {}).get("participatingcounties")
-        counties = []
-        for raw_county in raw_counties:
-            name, _id, version, last_updated = raw_county.split("|")[0:4]
-            counties.append(CountyIdentifier(_id, name, version, last_updated))
-        return counties
-
     # https://results.enr.clarityelections.com//GA//105369/270988/reports/detailxml.zip
-    def get_county_results(self, statepostal, county, **kwargs):
-        return self.get_detail_xml(f"{statepostal}/{county.name}/{county.id}/{county.version}")
+    def get_county_results(self, statepostal, county_name, county_id, county_version, **kwargs):
```

Comment:
Tiny pet peeve across this repo: we use `statepostal` everywhere, but in our other repos we usually use `state`. (This just forces me to copy one key into the other when preparing the fetcher params, so it's not the end of the world.)

The same thing happens for `electionid` / `election_id` below — it might be _slightly_ easier to standardize across all the repos, but isn't the end of the world either